blog/_posts/2017-09-21-Our-IntoxiCODED-cloudy-summer.md

```diff
@@ -0,0 +1,82 @@
+---
+title: Our IntoxiCODED cloudy summer
```

Comment:
Hi, thanks for the PR. You have changed Gemfile.lock. Can you please revert that change? For the blogpost PR, it is preferrable to submit blog post related files only. There is a separate issue related to versions of dependency packages. You can have a look at it and can send a separate PR for the same, in case you want to :)